src/gui/mzroll/ligandwidget.cpp

```diff
@@ -586,6 +590,11 @@ void LigandWidget::resetColor()
 
 void LigandWidget::updateIsotopesAndAdducts()
 {
+    QApplication::processEvents();
+    _busyMessage->setText("Populating isotopologues and adducts…");
```

Comment:
@saifulbkhan  This notification is also received when we uncheck the isotopes or adducts option. Is this the expected behaviour? As message 'populating isotopologues and adducts' didn't make sense to me, when we uncheck either of the two.

Reply:
Oh good point. I will substitute an appropriate message for such cases.